pom.xml

```diff
@@ -48,7 +47,6 @@
 			<id>runnable</id>
 			<properties>
 				<package.file>${project.build.directory}/${project.artifactId}.jar</package.file>
```

Comment:
delete `<package.file>${project.build.directory}/${project.artifactId}.jar</package.file>` and keep `<packaging.type>runnable</packaging.type>`